### ISSUE TITLE
Client Certificate

### ISSUE THREAD

**Issue by immanuelbaker** [MicrosoftDocs/windowsserverdocs#3143]
When talking about the client certificates, does the following "You must import the certificate to all computers that will communicate with the WSUS server. This includes all client computers, downstream servers, and computers that run the WSUS Administration Console. The certificate should be imported into the local computer Trusted Root CA store or into the Windows Server Update Service Trusted Root CA store."
... mean the certificate of the certification authority (CA) should be imported into the local computer Trusted Root CA store, or is it  the certificate used in the site binding for the SSL port? In my testing it looks like this only needs to be the certificate of the certification authority?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 80868bcd-a401-60a3-16b1-7817f4320dd1
* Version Independent ID: a7e9dc84-915a-9895-cf0d-0a0e89a062fe
* Content: [Step 2 - Configure WSUS](https://docs.microsoft.com/en-us/windows-server/administration/windows-server-update-services/deploy/2-configure-wsus#feedback)
* Content Source: [WindowsServerDocs/administration/windows-server-update-services/deploy/2-configure-wsus.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-server-update-services/deploy/2-configure-wsus.md)
* Product: **windows-server-threshold**
* Technology: **manage-wsus**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by timmmay** [comment]
You are correct, and I've just confirmed it myself.  The steps we took were as follows:

- internal CA already issuing certificates to clients and has it's own cert in the Trusted Root Authorities
- a new certificate was generated on the CA for the WSUS server name
- imported certificate on WSUS server and used it for the SSL site binding in IIS
- updated the virtual directories to required SSL as per this article
- ran the command: wsusutil cibfugyressl servername
- confirmed SSL comms to WSUS server all ok from client PC's

**Comment by IngridAtMicrosoft** [comment]
Thanks for your dedication to our documentation. If we have been unable to review your issue in a timely manner, we sincerely apologize for the delayed response. We have captured the informtion you have provided for review. We are closing this issue for now, but if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close
